CRM/Contribute/BAO/ManagePremiums.php

```diff
@@ -84,7 +84,7 @@ public static function setIsActive($id, $is_active) {
    *
    * @return CRM_Contribute_DAO_Product
    */
-  public static function add(&$params, $ids) {
+  public static function add(&$params, $ids = []) {
```

Comment:
outdated now we have CRM_Contribute_BAO_Product - we shouldn't need to change this?